main.py

```diff
@@ -308,7 +337,9 @@ async def on_message(self, message):
             content = message.content.lstrip(config.prefix)
             if not content:
                 return
-
+            if str(content).startswith('!random'):
```

Comment:
Create commands with the `bot.command` decorator, specifying the name there, rather than here.